### ISSUE TITLE
[BUG] Integration test is flaky

### ISSUE THREAD

**Issue by connorsmith256** [wasmCloud/wash-archive#478]
**Describe the bug**
`can_stop_detached_host` seems to be flaky, failing ~50% of the time with nextest

**Example in GHA**
https://github.com/wasmCloud/wash/actions/runs/4441870676/jobs/7797463153

I've reproduced this locally 3x in a row while running `make test-integration`. Not sure I can get it to pass locally at all

**Comment by ricochet** [comment]
The following tests consistently fail for me on `main` including `can_stop_detached_host `.

```bash
--- TRY 3 STDERR:        wash-cli::integration_up integration_up_can_start_wasmcloud_and_actor ---

{
  "error": "Could not start NATS server, a process is already listening on 127.0.0.1:5893",
  "success": false
}
thread 'integration_up_can_start_wasmcloud_and_actor' panicked at 'assertion failed: status.success()', tests/integration_up.rs:28:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

------------
     Summary [  46.890s] 29 tests run: 26 passed (1 flaky), 3 failed, 0 skipped
   FLAKY 2/3 [   7.906s] wash-cli::integration_inspect integration_inspect_cached
  TRY 3 FAIL [   1.101s] wash-cli::integration_up can_stop_detached_host
  TRY 3 FAIL [   0.665s] wash-cli::integration_up doesnt_kill_unowned_nats
  TRY 3 FAIL [   0.040s] wash-cli::integration_up integration_up_can_start_wasmcloud_and_actor
```

**Comment by brooksmtownsend** [comment]
Fixed by #520

**Comment by connorsmith256** [comment]
Resolved by #520

**Comment by connorsmith256** [comment]
Resolved by #520
